File: ip-messaging/rest/credentials/update-credentials/update-credentials.6.x.py
# Download the Python helper library from twilio.com/docs/python/install
import os
from twilio.rest import Client

# Initialize the client
# To set up environmental variables, see http://twil.io/secure
account = os.environ['TWILIO_ACCOUNT_SID']
token = os.environ['TWILIO_AUTH_TOKEN']
client = Client(account, token)

credential = client.chat \
    .credentials("CRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX") \
    .update(friendly_name="MyCredential", api_key=os.environ['TWILIO_API_KEY_SECRET'])

print(credential.friendly_name)

File: ip-messaging/rest/credentials/update-credentials/update-credentials.6.x.rb
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']
client = Twilio::REST::Client.new(account_sid, auth_token)

# Update the credential
response = client.chat
                 .v2.credentials('CRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                 .fetch.update(friendly_name: 'friendly_name')
puts response